scapy/layers/tls/automaton_cli.py

```diff
@@ -188,8 +202,49 @@ def INIT_TLS_SESSION(self):
             self.advertised_tls_version = default_version
 
         if s.advertised_tls_version >= 0x0304:
+            # For out of band PSK, the PSK is given as argument
+            # to the automaton
             if self.tls13_psk_secret:
                 s.tls13_psk_secret = binascii.unhexlify(self.tls13_psk_secret)
+
+            # For resumed PSK, the PSK is computed from
+            if self.session_ticket_file_in:
+                with open(self.session_ticket_file_in, 'rb') as f:
+
+                    resumed_ciphersuite_len = struct.unpack("B", f.read(1))[0]
+                    s.tls13_ticket_ciphersuite = \
+                        struct.unpack("!H", f.read(resumed_ciphersuite_len))[0]
+
+                    ticket_nonce_len = struct.unpack("B", f.read(1))[0]
+                    # XXX add client_session_nonce member in tlsSession
+                    s.client_session_nonce = f.read(ticket_nonce_len)
+
+                    client_ticket_age_len = struct.unpack("!H", f.read(2))[0]
+                    tmp = f.read(client_ticket_age_len)
+                    s.client_ticket_age = struct.unpack("!I", tmp)[0]
+
+                    client_ticket_age_add_len = struct.unpack("!H", f.read(2))[0]  # noqa: E501
```

Comment:
Will importing unpack with `from struct import unpack` remove the need of the `noqa` tag ?